### ISSUE TITLE
can add the option to export offline html?

### ISSUE THREAD

**Issue by HuangShumin** [markmap/markmap-vscode#44]
the offline html contains css/javascript all in one single html file.

**Comment by HuangShumin** [comment]
404
File not found

The site configured at this address does not contain the requested file.

If this is your site, make sure that the filename case matches the URL.
For root URLs (like http://example.com/) you must provide an index.html file.


This is the error when I embed all assets.

**Comment by gera2ld** [comment]
Did you try the `embedAssets` option? The usage is the same as in https://markmap.js.org/repl

**Comment by HuangShumin** [comment]
yes, if the embedAssets:true option is set, the `Download as interactive HTML' will be shown as 404 when it's opened.

**Comment by gera2ld** [comment]
So this is an issue for the website rather than the VSCode extension?
